docs/api/wanchain.md

```diff
@@ -1,5 +1,255 @@
+| deploy.tokenTransferData.receiverId | A `string` representing the Wanchain account address which will receive the value.
+| deploy.tokenTransferData.value | A `string` representing the value amount that will be transfered.
+
+**Result:**
+
+An instance of the same mutation class.
+
+**Example:**
+
+```ts
+// arbitrary data
+const deploy: Deploy = {
+  makerId: '0xb86051eCDe694d87D8f8f14B46557Db3704dbB4b',
+  takerId: '0x2B8C7FEf8f44fFfE07009B37fb7a031860B191a5',
+  seed: 1535113220,
+  expiration: Date.now() + 60 * 60 * 24, // 1 day
+  assetLedgerData: {
+    name: 'test',
+    symbol: 'TST',
+    uriBase: 'https://base.com/',
+    schemaId: '0x9c22ff5f21f0b81b113e63f7db6da94fedef11b2119b4088b89664fb9a3cb658',
+    capabilities: [AssetLedgerCapability.TOGGLE_TRANSFERS, AssetLedgerCapability.DESTROY_ASSET],
+    owner: '0xb86051eCDe694d87D8f8f14B46557Db3704dbB4b',
+  },
+  tokenTransferData: {
+    ledgerId: '0x09f4C86B43FD562E4A4B6627029Ff8Dff1c3cEA2',
+    receiverId: '0x2B8C7FEf8f44fFfE07009B37fb7a031860B191a5',
+    value: '50000',
+  },
+};
+
+// perform mutation
+const mutation = await deployDateway.cancel(deploy);
+```
+
+**See also:**
+
+[claim](#claim), [perform](#perform)
+
+### claim(deploy)
+
+An `asynchronous` class instance `function` which cryptographically signes the provided `deploy` and returns a signature.
+
+::: warning
+This operation must be executed by the maker of the deploy.
+:::
+
+**Arguments:**
+
+| Argument | Description
+|-|-
+| deploy.assetLedgerData.capabilities | [required] An array of `AssetLedgerCapability` representing the capabilities of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.name | [required] A `string` representing the name of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.owner | [required] A `string` representing the address of owner of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.schemaId | [required] A `string` representing the schemaId of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.symbol | [required] A `string` representing the symbol of the `AssetLedger` that will be deployed.
+| deploy.assetLedgerData.uriBase | [required] A `string` representing the uriBase of the `AssetLedger` that will be deployed.
+| deploy.expiration | [required] An `integer` number representing the timestamp in milliseconds at which the deploy expires and can not be performed any more.
+| deploy.makerId | [required] A `string` representing the Wanchain account address which makes the deploy. It defaults to the `accountId` of a provider.
+| deploy.seed | [required] An `integer` number representing the unique deploy number.
+| deploy.takerId | A `string` representing the Wanchain account address which will be able to perform the deploy on the blockchain. This account also pays for the gas cost.
+| deploy.tokenTransferData.ledgerId | A `string` representing the value ledger address of which value will be tranfered.
```

Comment:
| deploy.tokenTransferData.ledgerId | A `string` representing the value ledger address from which the value will be transferred.